tests/backend/test_organization_stats.py

```diff
@@ -0,0 +1,29 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import pytest
+from uuid import uuid4
+
+from tests.backend.common import vlob_create, block_create
+from parsec.core.backend_connection import BackendConnectionError
+
+
+@pytest.mark.trio
+async def test_organization_stats(running_backend, alice_backend_sock, alice_core, bob_core, realm):
+    organization_stats = await alice_core.get_organization_stats()
+    assert organization_stats.users == 3
+    assert organization_stats.data_size == 0
+    initial_metadata_size = organization_stats.metadata_size
+
+    # Create new metadata
+    await vlob_create(alice_backend_sock, realm_id=realm, vlob_id=uuid4(), blob=b"1234")
+    organization_stats = await alice_core.get_organization_stats()
+    assert organization_stats.metadata_size == initial_metadata_size + 4
+
+    # Create new data
+    await block_create(alice_backend_sock, realm_id=realm, block_id=uuid4(), block=b"1234")
+    organization_stats = await alice_core.get_organization_stats()
+    assert organization_stats.data_size == 4
+
+    # Bob is not admin, it should fail
+    with pytest.raises(BackendConnectionError):
+        await bob_core.get_organization_stats()
```

Comment:
Would be good to also test the stats are isolated between organizations (so using `otheralice_core` fixture we end up with different stats, note `otheralice_core` fixture doesn't exists yet but is trivial to implement from `otheralice` and `core_factory` fixtures ^^)

Reply:
done